testserver.go

```diff
@@ -21,31 +18,14 @@ func (h *thriftHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	thriftHandler(w, r)
 }
 
-func initThriftTestServer(cfg *config.Config, handler cli_service.TCLIService) *httptest.Server {
+func initThriftTestServer(handler cli_service.TCLIService) *httptest.Server {
 
 	// endpoint := fmt.Sprintf("%s:%d", cfg.Host, cfg.Port)
 	tcfg := &thrift.TConfiguration{
-		TLSConfig: cfg.TLSConfig,
+		TLSConfig: nil,
 	}
 
-	var protocolFactory thrift.TProtocolFactory
-	switch cfg.ThriftProtocol {
-	case "compact":
-		protocolFactory = thrift.NewTCompactProtocolFactoryConf(tcfg)
-	case "simplejson":
-		protocolFactory = thrift.NewTSimpleJSONProtocolFactoryConf(tcfg)
-	case "json":
-		protocolFactory = thrift.NewTJSONProtocolFactory()
-	case "binary":
-		protocolFactory = thrift.NewTBinaryProtocolFactoryConf(tcfg)
-	case "header":
-		protocolFactory = thrift.NewTHeaderProtocolFactoryConf(tcfg)
-	default:
-		panic(fmt.Errorf("invalid protocol specified %s", cfg.ThriftProtocol))
-	}
-	if cfg.ThriftDebugClientProtocol {
-		protocolFactory = thrift.NewTDebugProtocolFactoryWithLogger(protocolFactory, "client:", thrift.StdLogger(nil))
-	}
+	protocolFactory := thrift.NewTBinaryProtocolFactoryConf(tcfg)
```

Comment:
Curious - will we need the other protocols?

Reply:
not for the test server, I guess :-)